src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -187,6 +186,14 @@ function ComposerWithSuggestions({
         RNTextInputReset.resetKeyboardInput(findNodeHandle(textInputRef.current));
     }, [textInputRef]);
 
+    const debouncedSaveReportComment = useMemo(
```

Comment:
@DylanDylann Why are we using useMemo here?

Reply:
useMemo is used to memoize (or cache) the result of the _.debounce function

Reply:
Ok. I did some googling and seems useMemo would go efficiently with debounce. So I am fine with this.